onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -2039,6 +2039,8 @@ def aten_expand(self: TTensor, size: TInt) -> TTensor:
     """expand(Tensor(a) self, SymInt[] size, *, bool implicit=False) -> Tensor(a)"""
 
     size = op.Cast(size, to=INT64.dtype)
+    # To support -1 dim.
+    size = op.Abs(size)
```

Comment:
I understand -1 becomes 1. Was that intended or was I missing something?

Reply:
Only -1 becomes 1. Do you think comment is redundant?